ros2controlcli/README.rst

```diff
@@ -0,0 +1,3 @@
+Read the `Docs`_ for example usage and command line arguments.
```

Comment:
You can do either this or does it work with a symlink? So we have only one physical file? `ln -s` command in terminal.

Reply:
I first thought about symlink as well, but isn't it a bit problematic when the codebase should support various filesystems/OSes? Or has it improved in recent years?

Also tried `.. include`, but that github markup does not support includes.